ci-docker-images/oci-volume-provisioner-system-test/Makefile

```diff
@@ -12,17 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER_REGISTRY ?= iad.ocir.io
+OCIRREGISTRY ?= iad.ocir.io
+OCIRTENANCY ?= oracle
 DOCKER_IMAGE_NAME ?= oci-volume-provisioner-system-test
 DOCKER_IMAGE_TAG ?= 1.0.2
-
-.PHONY: build
+.PHONY: build 
 build:
-	docker build --build-arg http_proxy=${http_proxy} \
-		         --build-arg https_proxy=${https_proxy} \
-				 -t ${DOCKER_REGISTRY}/${DOCKER_REGISTRY_USERNAME}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} .
+	docker build -t $(OCIRREGISTRY)/$(OCIRTENANCY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) .
```

Comment:
Not sure if we can remove the proxies here, unless we only want to run this from a mac?